src/thumbnailer.cc

```diff
@@ -39,16 +39,81 @@ Thumbnailer::~Thumbnailer() { WebPAnimEncoderDelete(enc_); }
 Thumbnailer::Status Thumbnailer::AddFrame(const WebPPicture& pic,
                                           int timestamp_ms) {
   // Verify dimension of frames.
-  if (!frames_.empty() && (pic.width != (frames_[0].pic).width ||
-                           pic.height != (frames_[0].pic).height)) {
+  if (!frames_.empty() && (pic.width != frames_[0].pic.width ||
+                           pic.height != frames_[0].pic.height)) {
     return kImageFormatError;
   }
   WebPConfig new_config;
   WebPConfigInit(&new_config);
```

Comment:
this call can fail in case of library version mismatch. An assert could be useful.